Neos.Flow/Classes/Property/TypeConverter/ObjectConverter.php

```diff
@@ -40,14 +40,14 @@
 class ObjectConverter extends AbstractTypeConverter
 {
     /**
-     * @var integer
+     * @var string
      */
-    const CONFIGURATION_TARGET_TYPE = 3;
+    const CONFIGURATION_TARGET_TYPE = 'targetType';
```

Comment:
Why this unrelated change? Because Psalm was complaining about the new usage of `getConfigurationValue(..., ::CONFIGURATION_...)` from above and all other converter options already are strings. Thus this type mismatch is fixed, we can reduce psalm baseline and we can theoretically even add an argument typehint to the method.

Reply:
@bwaidelich You suggested we change this back to make this less breaking. While you're right about reducing breakiness, I do think having those converter options `int` while others are `string` is in fact bad code and should be fixed (though we could move this to another PR). `get/setConfigurationValue(...)` should only accept `string` for the second argument (=config option name)

Reply:
I still think that it should at least go into a separate PR. I could imagine that folks use this in forms